backend/src/components/permissionUtils.js

```diff
@@ -154,6 +154,31 @@ function checkSdcSchoolCollectionAccess(req, res, next) {
   }
 }
 
+function loadInstituteCollection(req, res, next) {
+  if (req.session.activeInstituteType === 'SCHOOL') {
+    return loadSdcSchoolCollection(req, res, next);
+  } else {
+    return loadSdcDistrictCollection(req, res, next);
+  }
+}
+
+function checkInstituteCollectionAccess(req, res, next) {
```

Comment:
Will there be a case when a district user is trying to access a school collection?

Reply:
Yes, district user can access school collection